### PR TITLE
Oclif command examples

### PR DESCRIPTION
**Pull request by devinsag**
please see https://gitlab.com/architect-io/architect-cli/-/issues/481

### CODE REVIEW

File: src/commands/dev.ts [hunk 1]
```diff
@@ -23,6 +23,11 @@ export default class Dev extends BaseCommand {
 
   static description = 'Run your stack locally';
 
+  static examples = [
+    'architect dev ./myComponent/architect.yml',
+    'architect dev --port=81 --browser --debug --secret-file=./myComponent/mySecrets.yml ./myComponent/architect.yml',
```
Comment (mueschm) [inline src/commands/dev.ts:28]: Browser is set to true by default. This should be `--no-browser` to show a change in functionality.

File: src/commands/platforms/destroy.ts [hunk 2]
```diff
@@ -8,7 +8,10 @@ import BaseCommand from '../../base-command';
 export default class PlatformDestroy extends BaseCommand {
   static aliases = ['platforms:deregister', 'platform:destroy', 'platforms:destroy'];
   static description = 'Deregister a platform from Architect';
-
+  static examples = [
+    'architect platform:destroy --account=myAccount architect',
+    'architect platforms:deregister --account=myAccount --type=kubernetes --auto-approve --force architect',
```
Comment (mueschm) [inline src/commands/platforms/destroy.ts:13]: There does not seem to be a flag `--type=kubernetes` for this command.

**Review by mueschm — CHANGES_REQUESTED** [review]

File: src/commands/dev.ts (hunk 1 above)
Comment (devinsag) [reply src/commands/dev.ts]: updated to use --no-browser

File: src/commands/platforms/destroy.ts (hunk 2 above)
Comment (devinsag) [reply src/commands/platforms/destroy.ts]: removed --type from platforms:deregister examples

File: src/commands/components/index.ts [hunk 3]
```diff
@@ -17,7 +17,11 @@ interface Component {
 export default class Components extends BaseCommand {
   static aliases = ['components', 'components:search', 'component:search', 'component:search'];
   static description = 'Search components you have access to';
-
+  static examples = [
+    'architect components',
+    'architect components --account=myAccount',
```
Comment (tjhiggins) [inline src/commands/components/index.ts:22]: Account names need to be lowercase - have this match our regex. Same with component.

```
architect components --account=myaccount
```

File: src/commands/components/versions.ts [hunk 4]
```diff
@@ -12,7 +12,10 @@ interface ComponentVersion {
 export default class ComponentVersions extends BaseCommand {
   static aliases = ['component:versions', 'component:version'];
   static description = 'Search component versions of a particular component';
-
+  static examples = [
+    'architect component:versions myComponentName',
```
Comment (tjhiggins) [inline src/commands/components/versions.ts:16]: lets be consistent in naming `mycomponent` or `mycomponentname`

**Review by tjhiggins — CHANGES_REQUESTED** [review]

File: src/commands/components/index.ts (hunk 3 above)
Comment (devinsag) [reply src/commands/components/index.ts]: renamed all example camelcase values to lowercase

File: src/commands/components/versions.ts (hunk 4 above)
Comment (devinsag) [reply src/commands/components/versions.ts]: renamed all example values {value}Name to {value} to be consistent

**Pull request merged**